app/jobs/create_institution_records_job.rb

```diff
@@ -16,7 +16,9 @@ def perform(institution_type_id)
       page += 1
 
       batch_insert_institutions = []
-      raw_institutions.each_with_object({}) do |raw_institution, institution|
+      raw_institutions.each do |raw_institution|
```

Comment:
If there was a bug, when fixing it, you should add a regression test, i.e., a test proving the bug is fixed.

Reply:
I checked it manually the bug has been fixed. Designing the test for is is going to take me a long time because I'd have to mock the entire RSPO api write the responses and check all fields. I don't have unlimited time to spend on done features. This feature won't be modified when it comes to business logic maybe with regard to fields that are added to the database.

Reply:
If this didn't work previously, it means it wasn't properly tested; if it wasn't properly tested, it was never done. It's not about spending time on done features; it's about making sure you're doing high-quality work. 
If you're making real requests from tests, all you have to do is to test how many records this job creates. If you want to stub requests, you don't have to mock the entire RSPO API, you can assume the page's size is 2 and the API contains 3 records. Both things are very easy.

Reply:
It worked perfectly fine before the refactoring was ordered. Yesterday during code review we agreed on this being merge ready provided that a few extremely minor things are fixed and they have been. I would prefer to spend time on new features that bring us closer to a goal of it being ready at the end of the week in a tangible way and not on applying refactoring to bugfixes for bugs resulting from refactoring and applying refactoring to refactoring to refactoring.

Reply:
Hmm, I think we misunderstood each other. What I'm asking for is to add a missing test, not to "apply refactoring to bugfixes for bugs resulting from refactoring and applying refactoring to refactoring to refactoring".
It'll probably be enough to change `expect(Institution.where(rspo_institution_type_id: 93).count).to be_positive` to `expect(Institution.where(rspo_institution_type_id: 93).count).to eq(number)`, where `number` is an actual number of created records. This'll probably take you less time than writing a comment.